### DISCUSSION TITLE
Resend verify email

### DISCUSSION THREAD

**Discussion by Lorentzo92** [payloadcms/payload#7701] [Q&A]
### Describe the Bug

I am not able to trigger the resend of the "verify email" email.

### To Reproduce

use standard payload latst version

### Payload Version

^2.26.0

**Comment by jmikrut** [comment]
Hey @Lorentzo92 — do you have your email configuration set up correctly? This is a problem with your email configuration and not with Payload, so I'd double-check your setup. Are you able to trigger any other emails such as forgot password, etc?

**Reply by Lorentzo92** [reply]
Hi James, thanks for the fast reply. I was not clear enough. The email is sent correctly and the forgot password email as well. I am wondering if it is possible to re-trigger the send of the verify email, if for some scenarios the user didnt receive the first email, it might happen often.

Thank You,
Lorenzo

**Reply by tomcatliu2016** [reply]
It is not that hard to customise one, but I would like to see it done natively by Payload!

**Comment by auduongtuan** [comment]
Any updates on this? I wish to have a resend verify email feature 🥲

**Comment by gCardinal** [comment]
Also adding my voice that this would be nice if it was baked in.

**Comment by matthijs166** [comment]
Plus one for this feature, now we build an custom endpoint to handle this.

Maybe an idea to add this as an option to the verify endpoint with an default cooldown of 5 minutes?
https://payloadcms.com/docs/authentication/operations#verify-by-email
